test/secrets_test.rb

```diff
@@ -45,6 +45,7 @@ def response_body(body)
   end
 
   around do |test|
+    ENV["testing"] = "true"
```

Comment:
makes little sense in an around if it is not reset ... move to the base level ...